pypeit/deprecated/processimages.py

```diff
@@ -599,7 +599,7 @@ def build_rn2img(self, trim=True):
         if trim:
             datasec_img = procimg.trim_frame(datasec_img, datasec_img < 1)
         detector = self.spectrograph.detector[self.det-1]
-        self.rn2img = procimg.rn_frame(datasec_img, detector['gain'], detector['ronoise'], numamplifiers=detector['numamplifiers'])
+        self.rn2img = procimg.rn_frame(datasec_img, detector['gain'], detector['ronoise'])
```

Comment:
Revisit this based on the decision for `variance_frame` and `rn_frame` in a previous comment.